### PR TITLE
Rerendered the feedstock with version 1.0.

### LINKED ISSUE #12: AppVeyor is hanging everywhere but PY34
@pelson it seems that `conda-build-all` is suffering from the same problem as `obvious-ci`.

### PR DESCRIPTION
**Pull request by pelson**
An attempt to fix #12.

### CODE REVIEW

**Comment by pelson** [comment]
😢 
 
```
The following NEW packages will be INSTALLED:
    anaconda-client: 1.4.0-py27_0   defaults   
    clyent:          1.2.1-py27_0   conda-forge
    conda:           4.1.6-py27_0   defaults   
    conda-build:     1.21.5-py27_0  defaults   
    conda-build-all: 0.13.1-py27_0  local      
    conda-env:       2.5.1-py27_0   defaults   
    funcsigs:        1.0.2-py27_0   conda-forge
    gitdb:           0.6.4-py27_0   conda-forge
    gitpython:       1.0.1-py27_0   conda-forge
    jinja2:          2.8-py27_0     conda-forge
    markupsafe:      0.23-py27_0    conda-forge
    menuinst:        1.4.1-py27_0   defaults   
    mock:            2.0.0-py27_0   conda-forge
    patch:           2.5.9-1        defaults   
    pbr:             1.10.0-py27_0  defaults   
    pip:             8.1.2-py27_0   defaults   
    psutil:          4.3.0-py27_0   defaults   
    pycosat:         0.6.1-py27_0   conda-forge
    python:          2.7.12-0       defaults   
    python-dateutil: 2.5.2-py27_0   conda-forge
    pytz:            2016.3-py27_0  conda-forge
    pyyaml:          3.11-py27_0    conda-forge
    requests:        2.10.0-py27_0  conda-forge
    ruamel_yaml:     0.11.7-py27_0  defaults   
    setuptools:      23.0.0-py27_0  conda-forge
    six:             1.10.0-py27_0  conda-forge
    smmap:           0.9.0-py27_0   conda-forge
    vs2008_runtime:  9.00.30729.1-2 defaults   
    wheel:           0.29.0-py27_0  defaults   
    yaml:            0.1.6-vc9_0    conda-forge [vc9]
[conda] C:\conda\conda-bld\test-tmp_dir>call activate _test 
******  B A T C H   R E C U R S I O N  exceeds STACK limits ******
Recursion Count=699, Stack Usage=90 percent
******       B A T C H   PROCESSING IS   A 
```

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ requirements:
     - setuptools
     - gitpython
     - conda >4
-    - conda-build >=1.20.2
+    - conda-build >=1.21.5
```
Comment (jakirkham) [inline recipe/meta.yaml]: Should this be 1.21.7?

Comment (jakirkham) [inline recipe/meta.yaml]: Looking at this [line]( https://github.com/SciTools/conda-build-all/blob/v0.13.2/conda-build-all.recipe/meta.yaml#L22 ).

**Comment by jakirkham** [comment]
One question, otherwise LGTM.

File: recipe/meta.yaml (hunk 1 above)
Comment (pelson) [inline recipe/meta.yaml]: 😉  thanks @jakirkham

**Pull request merged**